addons/crm/models/crm_lead.py

```diff
@@ -506,6 +508,11 @@ def _compute_recurring_revenue_monthly(self):
         for lead in self:
             lead.recurring_revenue_monthly = (lead.recurring_revenue or 0.0) / (lead.recurring_plan.number_of_months or 1)
 
+    @api.depends('recurring_revenue', 'probability')
+    def _compute_recurring_revenue_prorated(self):
```

Comment:
Would put this compute method after "_compute_recurring_revenue_monthly_prorated", so that it follows the same order as the fields.